src/main/io/ledstrip.c

```diff
@@ -484,20 +478,32 @@ static void applyLedFixedLayers()
         }
 
         if (ledGetOverlayBit(ledConfig, LED_OVERLAY_THROTTLE)) {
-            hOffset += scaledAux;
+			if (rcData[THROTTLE] < (PWM_RANGE_MIN + PWM_RANGE_MAX)/2) //if below average
```

Comment:
I apparently rewrote the ledstrip.c from 3.1.7 rather than from master, that's why it didn't work. But my downloaded clone of master doesn't build properly

/Library/Developer/CommandLineTools/usr/bin/make -j ./obj/betaflight_3.2.0_NAZE.hex
arm-none-eabi-objcopy -O ihex --set-start 0x8000000 obj/main/betaflight_NAZE.elf obj/betaflight_3.2.0_NAZE.hex
make[1]: arm-none-eabi-objcopy: No such file or directory
make[1]: *** [obj/betaflight_3.2.0_NAZE.hex] Error 1
make: *** [hex] Error 2